scripts/run-blackbox-tests.sh

```diff
@@ -26,7 +31,27 @@ docker exec montagu_db_1 psql -U vimc -d montagu -c \
        "CREATE USER MAPPING FOR vimc SERVER montagu_db_annex OPTIONS (user 'vimc', password 'changeme');"
 # -------------------------------------------------------------
 
-# Build and image that can run blackbox tests
+
+ORDERLY_IMAGE="vimc/orderly:master"
```

Comment:
It looks like almost all this logic this is duplicated from ./scripts/start-orderly-web - can that script be re-used here?

Reply:
They're a little bit different because in the blackbox tests, orderlyweb is getting pulled up as part of the docker compose configuration, but I'm pulling out the common stuff of orderlyweb dependencies into a shared script/